bg/lessons/advanced/concurrency.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Concurrency
+category: advanced
+order: 4
+lang: en
```

Comment:
We unfortunately can't have these `lang: en` pages in here, they create issues with the generated site.